README.md

```diff
@@ -163,7 +173,7 @@ const main = ([p, s]: [parameter,storage]): ret =>
      });
 ```
 
-Explanations :
+Explanations:
 
 - the first line `#import "@ligo/fa/lib/fa2/nft/NFT.jsligo" "NFT"` imports the ligo FA library
```

Comment:
![image](https://user-images.githubusercontent.com/45454460/211520780-62e218a7-433e-4e0f-99bd-1c1c6f8d6705.png)

This is obvious, anybody can understand we import it. You should explain **why** we import it.